src/partials/ImageInput.svelte

```diff
@@ -93,7 +93,7 @@
   <Modal mini onEscape={decline}>
     <Content>
       {#if loading}
-        <Spinner delay={0}>Uploading your media...</Spinner>
+        <Spinner delay={0}>Uploading your media using {settings.mediaprovider_url.split("/.well-known")[0] }</Spinner>
```

Comment:
use `displayUrl` instead

Reply:
Using displayUrl it looks like this:

![image](https://github.com/coracle-social/coracle/assets/125748180/bcd6b817-8548-45ea-8aab-066bf82639f3)

The other way it looks like this:

![image](https://github.com/coracle-social/coracle/assets/125748180/9d83f283-0163-493e-b203-19bf60419ac0)

Do you want me to add something like "displayHostname" in misc?

Reply:
Ah, you're right, I misread the regex. Yeah, that would be great.

Reply:
You have displayDomain in "misc", I didn't see it, I will use this one :)

Reply:
done